receiver/awscontainerinsightreceiver/internal/k8sapiserver/k8sapiserver.go

```diff
@@ -44,7 +44,7 @@ type eventBroadcaster interface {
 	StartLogging(logf func(format string, args ...any)) watch.Interface
 	// NewRecorder returns an EventRecorder that can be used to send events to this EventBroadcaster
 	// with the event source set to the given event source.
-	NewRecorder(scheme *runtime.Scheme, source v1.EventSource) record.EventRecorder
+	NewRecorder(scheme *runtime.Scheme, source v1.EventSource) record.EventRecorderLogger
```

Comment:
Fix for a breaking change: https://github.com/kubernetes/client-go/commit/3595e5242a2c3e8d7b114b32361b2226f173f40e#diff-35acc6015fded274d91d420a4def1d1530a6b1cf62c7cb9143a8da3419413305